src/client/refresh/vk/vk_image.c

```diff
@@ -336,49 +336,32 @@ VkResult QVk_CreateImage(uint32_t width, uint32_t height, VkFormat format, VkIma
 		imageInfo.pQueueFamilyIndices = queueFamilies;
 	}
 
-	VmaAllocationCreateInfo vmallocInfo = {
-		.flags = texture->vmaFlags,
-		.usage = memUsage
-	};
-
 	texture->sharingMode = imageInfo.sharingMode;
-	return vmaCreateImage(vk_malloc, &imageInfo, &vmallocInfo, &texture->image, &texture->allocation, &texture->allocInfo);
+	return image_create(&texture->resource, imageInfo, /*mem_properties*/ 0, /*mem_preferences*/ 0);
 }
 
 void QVk_CreateDepthBuffer(VkSampleCountFlagBits sampleCount, qvktexture_t *depthBuffer)
 {
 	depthBuffer->format = QVk_FindDepthFormat();
 	depthBuffer->sampleCount = sampleCount;
-	// On 64-bit builds, Intel drivers throw a warning:
```

Comment:
Did you check on an Intel GPU that this issue is not present with your code?

Reply:
I don't have Intel hardware, could some one check it?